lib/developer_portal/lib/liquid/template/view.rb

```diff
@@ -3,9 +3,17 @@ class Template
     class View < ActionView::Template
       attr_reader :record
 
-      def initialize(source, ident, handler, details)
-        source ||= '' # in case template is nil
-        super
+      def initialize(source = '', identifier, handler, details)
```

Comment:
what is this change for?

Reply:
See the log here: https://github.com/3scale/porta/commit/a6d7ae2238c256e7b79352f44a5adf334dcfc487

It's required because the initializer for the parent class changed, for instance now a `locals` param is mandatory, so we have to adapt to it.